src/otaclient/configs/_proxy_info.py

```diff
@@ -85,6 +85,14 @@ class ProxyInfo(BaseFixedConfig):
         f"http://127.0.0.1:{LOGGING_SERVER_GRPC_PORT}"
     )
 
+    external_cache_mnt_point: Optional[StrOrPath] = None
+    """The mount point of external cache storage device.
+    If None, no external cache device is used.
+    """
+
+    is_nfs_cache: bool = False
+    """Whether the external cache storage device is NFS."""
+
```

Comment:
This is a good point. From my understanding, in the current code, `external_cache_mnt_point` is used for both local and NFS caches, and the `is_nfs_cache flag` is used to distinguish between them.

I think it’s better not to change the existing local behavior. So, how about we:
  - Rename `external_cache_mnt_point` to `external_nfs_cache_mnt_point` and use it only for NFS mount points.
  - Remove the `is_nfs_cache flag` and instead judge NFS usage based on whether `external_nfs_cache_mnt_point` is `None` or not.

What do you think? cc: @Bodong-Yang